Cargo.toml

```diff
@@ -24,7 +24,7 @@ dirs = "4.0.0"
 indicatif = "0.16.2"
 inquire = "0.2.1"
 miette = { version = "4.2.1", features = ["fancy"] }
-reqwest = { version = "0.11.10", features = ["json"] }
+reqwest = { version = "0.11.10", features = ["json", "native-tls-vendored"] }
```

Comment:
I would much prefer to use rustls, it looks like there is another feature for that: 

https://github.com/seanmonstar/reqwest/blob/master/Cargo.toml#L40

Reply:
Done, also removed arm64 Windows build since rustls on crates.io doesn't support it yet.